common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -275,8 +277,11 @@ describe 'Problem', ->
               callable()
             done: (callable) ->
               callable()
-        spyOn @problem.submitButtonLabel, 'text'
+        spyOn(@problem.submitButtonLabel, 'text')
+        $('#input_example_1').val('test').trigger('input')
```

Comment:
It seems like this test is covering the same sort of things (with regard to enable/disable state) as the previous test point. Do we need 2 different test points? 

If it makes sense to keep a separate test around text changing (which I can certainly be on board with), I'd recommend removing the enable/disable checks from this test point, as the coverage should be in the test point that is specifically about enable/disable state.